front/src/views/Client/CancelFlight/Cancel.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <Dialog v-model:open="isVisible">
+    <DialogContent class="sm:max-w-md">
+      <DialogHeader>
+        <DialogTitle>Confirmar Cancelamento</DialogTitle>
+        <br>
+        <DialogDescription>
+          Tem certeza de que deseja cancelar esta reserva? Esta ação não pode ser desfeita.
+        </DialogDescription>
+      </DialogHeader>
```

Comment:
Evitar utilizar tag `<br> `apenas para dar espaçamento, ao invés disso utilize por exemplo o elemento css `gap`.
```suggestion
      <DialogHeader class="gap-x">
        <DialogTitle>Confirmar Cancelamento</DialogTitle>
        <DialogDescription>
          Tem certeza de que deseja cancelar esta reserva? Esta ação não pode ser desfeita.
        </DialogDescription>
      </DialogHeader>
```

Reply:
feito também